setup.py

```diff
@@ -3,6 +3,8 @@
 tf_requirements = ['tensorflow==1.15.0']
 torch_requirements = ['torch==1.4.0', 'torchtext==0.5.0']
 lightgbm_requirements = ['lightgbm==2.3.0']
+geoanalytics_requirements = ['networkx==2.4', 'Shapely~=1.7.0', 'geopandas==0.7.0']
```

Comment:
I think we can keep networkx in the general requirements, see also my comment on the graph stuff below